src/app.js

```diff
@@ -0,0 +1,65 @@
+import {FilterableTable} from './components/FilterableTable.react';
+
+import * as SharedUtils from './utils/SharedUtils';
+
+document.addEventListener('DOMContentLoaded', function(){
+  var searchObject = SharedUtils.parseUrlSearchString(),
```

Comment:
So a helper method that returns an object of configurations:

```Javascript
var domParser = function(domNode) {

  return {
    filterBarConfiguration: '',
    tableConfiguration: ''
  }
}
```

Put it in SetupUtils or SharedUtils or something and call it from app.js?